packages/web-components/fast-components/docs/design/localization.md

```diff
@@ -16,7 +16,7 @@ Many CSS layout properties like [flexbox](https://developer.mozilla.org/en-US/do
 **Example: Using `DirectionalStyleSheetBehavior`**
 ```ts
 import { css } from "@microsoft/fast-element";
-import { DirectionalStyleSheetBehavior } from "@microsoft/fast-foundation";
+import { DirectionalStyleSheetBehavior } from "@fluentui/web-components";
```

Comment:
While this is also currently exported from Fluent UI, I think for this package we should likely call out that it comes from `@microsoft/fast-components`.